examples/federated-database-query-limit/README.md

```diff
@@ -0,0 +1,50 @@
+# Example - MongoDB Atlas Federated Database Query Limit with Atlas clusters
+
+This project aims to provide an example of using [MongoDB Atlas Federated Database Query Limit](https://www.mongodb.com/docs/atlas/data-federation/overview/).
+
+
+## Dependencies
+
+* Terraform MongoDB Atlas Provider v1.10.0
+* A MongoDB Atlas account 
+
+```
+Terraform v1.10.0
```

Comment:
The [latest terraform version](https://releases.hashicorp.com/terraform/) is 1.4.6

Reply:
I think this is supposed to be the Atlas plugin version, not sure. @Zuhairahmed can you confirm?